src/viking.js

```diff
@@ -1,11 +1,118 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength){
+        this.health = health;
+        this.strength = strength;
+    }
+
+    attack() {
+        return this.strength
+    }
+
+    receiveDamage(damage){
+        this.health -= damage;
+   
+}
+};
+
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health, strength){
+        super('name');
```

Comment:
For the super we wanted to pass the information needed on the parent's constructor
```suggestion
        super(health, strength);
```